### PR TITLE
refresh incantations

### PR DESCRIPTION
**Pull request by refack**
I can never remember the exact CLI syntax, so it's nice to have precooked lines documented.

### CODE REVIEW

File: doc/jenkins-guide.md [hunk 1]
```diff
@@ -62,10 +62,10 @@ be run on the worker itself, after SSH-ing in and switching to the
 
 ##### Out of memory
 
-First, get statistics on running processes for the machine: `ps aux | grep node | grep -v -egrep -ejava`.
+First, get statistics on running processes for the machine: `ps aux | grep node`.
```
Comment (mhdawson) [inline doc/jenkins-guide.md]: just wondering why the java one does not make sense anymore.  The jenkins agents still runs with Java.

Comment (refack) [reply doc/jenkins-guide.md]: IMHO getting the stats of the `java` proc (or if there is none, or more then 1) is interesting. Also this is a simpler line...

Comment (mhdawson) [reply doc/jenkins-guide.md]: Ok I see now, that its a -v, so its adding not removing the java processes.  But in that case does removing the -egrep make sense?

Comment (refack) [reply doc/jenkins-guide.md]: We should hit those procs with the `node` in `ci.nodejs.org`, e.g.:
```
iojs       839     1  1 647319 174936 0 /usr/bin/java -Xmx128m -jar /home/iojs/slave.jar -jnlpUrl https://ci.nodejs.org
                                                                                                             ^^^^
```

**Comment by sam-github** [comment]
@AshCripps this might be useful to land after rebasing

**Comment by AshCripps** [comment]
@sam-github Ive rebased and correct the typos in the commit titles, want to review before I land?

**Comment by sam-github** [comment]
The commit title is still `refresh incantations`, I think it should be `doc: refresh debugging notes` or some other less arcane title.

Othewise LGTM, and I think a year and a half is long enough for this to land now.

**Pull request merged**